client_test.go

```diff
@@ -81,6 +81,7 @@ func TestGetVideo(t *testing.T) {
 	video, err := testClient.GetVideo(streamURL)
 	require.NoError(err)
 	require.NotNil(video)
+	require.NotEmpty(video.Thumbnails)
```

Comment:
Please also check at least one field of the first thumbnail.
You have removed the JSON-Tags from the thumnail struct. Does it still work?